action.sh

```diff
@@ -32,9 +32,15 @@ if [[ -z "$GITHUB_TOKEN" ]] || [[ -z "$GITHUB_PULL" ]]; then
     echo "WARNING: missing GITHUB_TOKEN or GITHUB_PULL, skipping git actions"
     SKIP_GIT=1
 fi
+if [[ -z "$REQUIRED_ROLES" ]]; then
+    echo "WARNING: missing REQUIRED_ROLES, consider adding them to the action config"
+fi
+
 
 DOCKER_CLEANUP=""
 
+ROLES_FILE_PATH="$ACTION_PATH/init_roles.sh"
```

Comment:
Is this still needed now that you moved it to the main script?

Reply:
Cheers, missed that.